test/test.cpp

```diff
@@ -37,6 +37,13 @@
 #define TEST_LOG(fmt, ...) \
     fprintf(stderr, "--- ");fprintf(stderr, fmt, __VA_ARGS__);
 
+
+template<>
```

Comment:
Is the template jc_test_print_value still in use or was it some temp code?

Reply:
This overload is still needed to make `ASSERT_EQ` (and friends) user friendly when asserting on uint64_t values.

Without overload:
`Expected: (actual_size) == (expected_size), actual: ? vs ?`

With overload:
`Expected: (actual_size) == (expected_size), actual: 2147479552 vs 4294967296`